pkg/api/norman/customization/cluster/validator.go

```diff
@@ -394,15 +400,41 @@ func (v *Validator) validateEKSConfig(request *types.APIContext, cluster map[str
 
 	// validation for creates only
 
```

Comment:
any reason this check is below public/private check, but above networking ?

Reply:
Public/private can be updated while the other networking fields cannot.